lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2765,6 +2763,9 @@ struct FIRCircuitParser : public FIRParser {
 
   SmallVector<DeferredModuleToParse, 0> deferredModules;
   ModuleOp mlirModule;
+
+  // Indentation level for 'module' when not used as an identifier
+  const unsigned moduleIndent = 2;
```

Comment:
There shouldn't be a hard coded indent.  I believe firrtl allows 1 space indent too, or 3, etc.